src/preferences/dialog/dlgprefreplaygaindlg.ui

```diff
@@ -246,6 +221,43 @@
     </widget>
    </item>
    <item>
+    <widget class="QGroupBox" name="groupBoxAnalyzis">
+     <property name="title">
+      <string>Analyzis</string>
```

Comment:
It should be American English: 
http://grammarist.com/spelling/analyse-analyze/

Just to verify: Is it "to analyze" but analysis? Strange.